### PR TITLE
Add test and docs for top level `mypy: disable-error-code` comment

### LINKED ISSUE #14750: # mypy: disable-error-code=... usage details not documented
**Documentation**

(Based on a SO question) To disable multiple error codes in a whole file, none of the "obvious" ways works.
Please document that feature in greater detail.

---

Not OK:

```
# mypy: disable-error-code=misc
# mypy: disable-error-code=attr-defined
```

```
# mypy: disable-error-code=misc
# mypy: disable-error-code=attr-defined
```

The problem here is that the last such line seems to overwrite alle previous ones. Don't know if it is a bug.

---

Also not working:

`# mypy: disable-error-code=misc, attr-defined`

`# mypy: disable-error-code=misc attr-defined`

Here the problem is the syntax.

---

The correct command uses quotes, I wouldn't guess that:

`# mypy: disable-error-code="misc, attr-defined"`

### PR DESCRIPTION
**Pull request by sobolevn**
I went with the smallest change possible.
@xitop, please, take a look. Does it make sense to you? :)

Closes #14750

### CODE REVIEW

**Review by hauntsaninja — APPROVED** [review]
This looks good, could you also update the error message here: https://github.com/python/mypy/blob/d5860707544bdb3e51442bf6bd97b971f8c01d61/mypy/fastparse.py#L494

**Comment by xitop** [comment]
> I went with the smallest change possible. @xitop, please, take a look. Does it make sense to you? :)

Yes, looks fine. Thank you for your work. But regarding the typing and using mypy I'm a beginner, so don't count on my opinion too much.

**Comment by sobolevn** [comment]
@xitop beginners provide the best feedback for the docs! Thank you for the report 👍

**Comment by sobolevn** [comment]
@hauntsaninja we now have smart error message, thanks for the suggestion :)

File: mypy/fastparse.py [hunk 1]
```diff
@@ -487,11 +487,12 @@ def translate_stmt_list(
             and self.type_ignores
             and min(self.type_ignores) < self.get_lineno(stmts[0])
         ):
-            if self.type_ignores[min(self.type_ignores)]:
```
Comment (hauntsaninja) [inline mypy/fastparse.py:490]: We can't use walrus just yet; we still support 3.7

Comment (sobolevn) [reply mypy/fastparse.py:490]: This is the first time I've used walrus in real code :)
Probably, the last time.

**Review by hauntsaninja — APPROVED** [review]

**Pull request merged**